Makefile

```diff
@@ -226,6 +226,9 @@ setup-config-prompts:
 	 workspace_dir=$${workspace_dir:-$(DEFAULT_WORKSPACE_DIR)}; \
 	 echo "workspace_base=\"$$workspace_dir\"" >> $(CONFIG_FILE).tmp
 
+	@read -p "Enter a password for the sandbox container: " ssh_password; \
```

Comment:
💭  can we allow users to set `persist_sandbox` first, and if they set it to false, we skip this one?

Reply:
That would complicate new users. Why don't they want persistent sandboxes?

https://github.com/OpenDevin/OpenDevin/pull/1998#issuecomment-2127170406
cc: @rbren

Reply:
Because:
1) I don't want to use opendevin again - I don't want to see opendevin leave any unused containers.
2) I am using E2B sandbox and this password thing is irrelevant
3) I don't want to remember a password or record my password anywhere

Reply:
2nd one is relevant. Updating.